x-pack/plugins/security_solution/public/resolver/view/panels/node_list.tsx

```diff
@@ -188,13 +219,30 @@ function NodeDetailLink({ name, item }: { name: string; item: ProcessTableView }
           )}
         </EuiBadge>
       ) : (
-        <>
+        <StyledButtonTextContainer>
           <CubeForProcess
             running={!isTerminated}
+            isOrigin={isOrigin}
             data-test-subj="resolver:node-list:node-link:icon"
           />
-          <span data-test-subj="resolver:node-list:node-link:title">{name}</span>
-        </>
+          <StyledProcessLabel color={descriptionText}>
+            {isOrigin && (
+              <div data-test-subj="resolver:node-list:node-link:origin" className="origin-label">
+                <span>
+                  {i18n.translate(
+                    'xpack.securitySolution.endpoint.resolver.panel.table.row.originLabel',
```

Comment:
Would you please remove 'endpoint' from the ID there?

Reply:
Yea, removed. I removed the endpoint references in all the translation ids in this file. Is there any follow up work that needs to be done to update the changes elsewhere?

Reply:
sounds good